scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -k <S3 key prefix>"
```

Comment:
I think `-k [S3 key prefix]` would indicate that you can call it with `-k` or `-k some_prefix`. Using that style of notation, we'd actually want `[-k <S3 key prefix>]` but I dunno if that's readable / understandable? But maybe that'd be sufficient, given that it's probably only ever Etcher developers who will see those messages? ;-)